containers/datalab/run.sh

```diff
@@ -65,6 +82,7 @@ fi
 docker run -it --entrypoint=$ENTRYPOINT \
   -p 127.0.0.1:8081:8080 \
   -v "$CONTENT/datalab:/content/datalab" \
+  $PYDATALAB_MOUNT_OPT \
```

Comment:
I think the build-step option becomes unnecessary once this is in, since this change sort of covers that other workflow. I'll let @qimingj chime in on this since he's the one who added it IIRC.